Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserEntityInspectorWidget.h

```diff
@@ -8,9 +8,16 @@
 #pragma once
 
 #if !defined(Q_MOC_RUN)
-#include <QDockWidget>
+#include <AzCore/std/containers/set.h>
 #include <AzToolsFramework/AssetBrowser/AssetBrowserBus.h>
-#include <AzCore/Asset/AssetCommon.h>
+#include <AzToolsFramework/AssetDatabase/AssetDatabaseConnection.h>
+#include <AzToolsFramework/AssetBrowser/Entries/ProductAssetBrowserEntry.h>
+#include <AzToolsFramework/AssetBrowser/Entries/SourceAssetBrowserEntry.h>
+#include <AzQtComponents/Components/Widgets/Card.h>
```

Comment:
nit: include order - 'AssetBrowser' should be before 'AssetDatabase'

Reply:
Good catch, fixed